install/empty_data.php

```diff
@@ -3451,6 +3458,11 @@ public function getEmptyData(): array
                 'notifications_id'         => '76',
                 'mode'                     => 'mailing',
                 'notificationtemplates_id' => 31,
+            ], [
+                'id'                       => 77,
+                'notifications_id'         => '77',
+                'mode'                     => 'mailing',
+                'notificationtemplates_id' => 32,
```

Comment:
IMHO, we could use the same template as for `add_followup`/`add_task`/`delete_followup`/`delete_task` events.
```suggestion
                'notificationtemplates_id' => 4,
```